cmake/external_onnx.cmake

```diff
@@ -22,6 +22,7 @@ include(ExternalProject)
 #------------------------------------------------------------------------------
 
 set(ONNX_VERSION 1.3.0)
+set(ONNX_OPSET_VERSION 7)
```

Comment:
This does not seem like the right place to hard-code the ONNX opset version.
We should be reporting the highest opset version that nGraph supports and this information should be taken from nGraph source code, this seems like a redundant second copy, which will only get out of date.